### DISCUSSION TITLE
Developer Meetings / Hackathons ?

### DISCUSSION THREAD

**Discussion by markdoerr** [emmo-repo/EMMO-python#591] [Q&A]
Are there regular developer EMMOntoPy developer meetings / hackathons ?

**Comment by francescalb** [comment]
There is a WG in EMMC for EMMOntoPy, but it is mostly us in SINTEF developing on it when needed. Are you interested in joining such a WG?

**Comment by markdoerr** [comment]
If I can manage, I would try to join - maybe I try to join one of the next meetings and I will see, if it is too specific or not.
I there there an  EMMO planned in the next future ?
